targets/targets.json

```diff
@@ -5092,6 +5092,67 @@
         "features": ["BLE"],
         "release_versions": ["2", "5"]
     },
+
+    "ARM_MUSCA_A1": {
+        "public": false,
+        "inherits": ["Target"],
+        "supported_toolchains": ["ARMC6", "GCC_ARM"],
+        "default_toolchain": "ARMC6",
+        "extra_labels": ["ARM_SSG", "MUSCA_A1"],
+        "forced_reset_timeout": 7,
+        "release_versions": ["5"]
+    },
+    "ARM_MUSCA_A1_NS": {
+        "inherits": ["NSPE_Target", "ARM_MUSCA_A1"],
+        "core": "Cortex-M33-NS",
+        "device_has_add": ["INTERRUPTIN", "LPTICKER", "SERIAL", "SLEEP", "USTICKER"],
+        "macros": [
+            "MBED_TZ_DEFAULT_ACCESS=1",
+            "__STARTUP_CLEAR_BSS",
+            "MBED_FAULT_HANDLER_DISABLED",
+            "CMSIS_NVIC_VIRTUAL",
+            "LPTICKER_DELAY_TICKS=1",
+            "MBED_MPU_CUSTOM",
+            "TFM_PSA_API",
+            "MBEDTLS_PSA_CRYPTO_C"
+        ],
+        "extra_labels_add": ["MUSCA_A1_NS", "PSA", "TFM"],
```

Comment:
PSA and TFM should be component ?

Reply:
no, they are targets, so build system will include `TARGET_PSA` and `TARGET_TFM` folders.

Reply:
For me, "feature" from "component" directory should be enabled with "component_add", not with "extra_labels-add"
I thought it was mbed-os philosophy ?

Reply:
I also question: `components/TARGET_PSA`. Why this is not `COMPONENT_PSA` ? I see components are inside `TARGET_MBED_SPM`.

Reply:
@0xc0170 @jeromecoutant We have no objection to changing it to **COMPONENT_**.
But this needs to be in a separated PR, as **TARGET_PSA** is with us since 5.11 release.

Reply:
@0xc0170 - I have been requesting the reverse, to move Target_PSA to target (as it is target specific code). It was not possible in 5.11 due to build system limitations. But seems possible after 5.12 release of updated tools

Reply:
@deepikabhavnani what exactly did you request?
Moving the entire **TARGET_PSA** folder into `mbed-os/targets/`?

Reply:
@jeromecoutant @0xc0170 @deepikabhavnani as @mikisch81 said the `components/TARGET_PSA` was brought in for the 5.11, so it is already mainlined. If we want to move or rename this folder (based on the conversation above, that sounds like a good plan anyway), can we move that discussion and work to a separate PR?

Reply:
@mikisch81 Can you capture this in the new issue please?

Reply:
👍

Reply:
@0xc0170 Has this already been captured as an issue? If so can you please link it? If not then I can create an issue.